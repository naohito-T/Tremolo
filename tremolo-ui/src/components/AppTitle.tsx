// import React from "react"; いらなくなったかも
import styled from 'styled-components';

// interface Props {
//   children: string;
// }

/** ここ直せ */
const Title: React.FC<any> = styled.h1`
  color: red;
  font-size: 50px;
`;

const AppTitle: React.FC = () => {
  return <Title>message</Title>;
};

export default AppTitle;
