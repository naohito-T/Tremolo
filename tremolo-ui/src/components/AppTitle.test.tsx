/**
 * @jest-environment jsdom
 */
import React from "react";
import { render } from "@testing-library/react";
import AppTitle from "./AppTitle";

test("AppTitle", () => {
  const { asFragment } = render(<AppTitle />);
  expect(asFragment()).toMatchSnapshot();
});
