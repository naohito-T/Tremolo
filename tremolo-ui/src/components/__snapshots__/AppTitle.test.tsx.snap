// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppTitle 1`] = `
<DocumentFragment>
  <h1
    class="sc-bczRLJ doLKWM"
  >
    message
  </h1>
</DocumentFragment>
`;
